bench/include/rust/Cargo.toml

```diff
@@ -60,4 +60,5 @@ lto = true
 opt-level = 3
 overflow-checks = false
 codegen-units = 1
-target-cpu = "native"
\ No newline at end of file
+target-cpu = "skylake"
+target-feature = "+avx2, +fma, -avx512f"
```

Comment:
these won't work, need to update RUSTFLAGS in bench_rust.yaml, they're supposed tobe set in global cargo config instead of project manifest